.github/workflows/publish.yaml

```diff
@@ -52,6 +52,12 @@ on:
           The git ref to read BCR templates (.bcr folder) rather than reading them from `tag_name`.
           Use this to republish a release whose templates had errors.
         type: string
+      attest:
```

Comment:
Add : "If your source archive does not include attestations from `https://github.com/bazel-contrib/.github/.github/workflows/release_ruleset.yaml`, you may want to skip this."

OR alternatively, we can fail the workflow if a matching attestation isn't found?

Reply:
Since we  *know* (for now) we're not allowed anything else, might as well fail early?

Reply:
Updated the description.